__tests__/app.test.js

```diff
@@ -38,3 +38,23 @@ describe("/api/categories", () => {
     });
   });
 });
+
+describe("/api/users", () => {
+  describe("GET", () => {
+    test("200: Should respond with an array of all users", () => {
+      return request(app)
+        .get("/api/users")
+        .expect(200)
+        .expect("Content-Type", "application/json; charset=utf-8")
+        .then(({ body }) => {
+          expect(Array.isArray(body)).toBe(true);
+          expect(body.length > 0).toBe(true);
```

Comment:
You can be way more specific about the length of the data array you expect back. The test data is re-seeded after each test and you have access to the data, it will be the same number of items each time

Reply:
✅